content/08.methods.md

```diff
@@ -9,10 +9,10 @@ We wrote the manuscript in markdown and tracked changes using git.
 Contributions were handled through GitHub, with individuals submitting "pull requests" to suggest additions to the manuscript.
 
 To facilitate citation, we [defined](https://github.com/greenelab/deep-review/blob/master/CONTRIBUTING.md#markdown) a markdown citation syntax.
-We supported citations to the following identifier types (in order of preference): DOIs, PubMed IDs, arXiv IDs, and URLs.
+We supported citations to the following identifier types (in order of preference): DOIs, PubMed Central IDs, PubMed IDs, arXiv IDs, and URLs.
 References were automatically generated from citation metadata by querying APIs to generate [Citation Style Language](http://citationstyles.org/) (CSL) JSON items for each reference.
 [Pandoc](http://pandoc.org/) and [pandoc-citeproc](https://github.com/jgm/pandoc-citeproc) converted the markdown to HTML and PDF, while rendering the formatted citations and references.
-In total, referenced works consisted of {{manuscript_stats.reference_counts.doi}} DOIs, {{manuscript_stats.reference_counts.pmid}} PubMed records, {{manuscript_stats.reference_counts.arxiv}} arXiv manuscripts, and {{manuscript_stats.reference_counts.url}} URLs (webpages as well as manuscripts lacking standardized identifiers).
+In total, referenced works consisted of {{manuscript_stats.reference_counts.doi}} DOIs, {{manuscript_stats.reference_counts.pmcid}} PubMed Central records, {{manuscript_stats.reference_counts.pmid}} PubMed records, {{manuscript_stats.reference_counts.arxiv}} arXiv manuscripts, and {{manuscript_stats.reference_counts.url}} URLs (webpages as well as manuscripts lacking standardized identifiers).
```

Comment:
Same question here, should we remove PubMed ID counts?  Or are there some that couldn't be converted to PMCIDs?